docs/api/countries.rst

```diff
@@ -0,0 +1,32 @@
+.. _countries:
+
+.. currentmodule:: nomenclature.countries
+
+A common list of countries
+==========================
+
+Having an agreed list of country names including a mapping to alpha-3 and alpha-2 codes
+(also know as ISO3 and ISO2 codes) is an important prerequisite for scenario analysis
+and model comparison.
+
+The :class:`nomenclature` package builds on the :class:`pycountry` package
+(`link <https://github.com/flyingcircusio/pycountry>`_) to provide a utility for country
+names based on the ISO 3166-1 standard.
+
+For consistency with conventions in the modelling community, several
+country names are shortened compared to ISO 3166-1 ,
+e.g. from "Bolivia, Plurinational State of" to "Bolivia".
```

Comment:
Maybe also put the full list here, maybe at the bottom.

Reply:
Added a link to the GitHub repo module (which will only work after merging the PR).